### PR TITLE
Added Clementine component doc.

### PR DESCRIPTION
**Pull request by jjmontesl**
**Description:**

Adds documentation for the Clementine Remote Player component (pull request to home-assistant).

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** 

https://github.com/home-assistant/home-assistant/pull/5877

### CODE REVIEW

**Comment by jjmontesl** [comment]
Shall this be added to some index of components?

Also, shall I check if Clementine icon can be included? how is this managed?

File: source/_components/media_player.clementine.markdown [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+sharing: true
+footer: true
+logo: lg.png
+ha_category: Media Player
+ha_iot_class: "Local Poll"
+ha_release: "0.38"
+ha_iot_class: "Local Polling"
+---
+
+The `lg_netcast` platform allows you to control a [Clementine Music Player](https://www.clementine-player.org).
```
Comment (fabaff) [inline source/_components/media_player.clementine.markdown]: Stale platform name.

File: source/_components/media_player.clementine.markdown [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+description: "Instructions how to integrate Clementine Music Player within Home Assistant."
+date: 2017-02-11 17:15
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: lg.png
+ha_category: Media Player
+ha_iot_class: "Local Poll"
+ha_release: "0.38"
```
Comment (fabaff) [inline source/_components/media_player.clementine.markdown]: 0.38 will be releases in a couple of hours. Will be at least 0.39.

File: source/_components/media_player.clementine.markdown [hunk 3]
```diff
@@ -0,0 +1,42 @@
+---
+layout: page
+title: "Clementine Music Player"
+description: "Instructions how to integrate Clementine Music Player within Home Assistant."
+date: 2017-02-11 17:15
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: lg.png
```
Comment (fabaff) [inline source/_components/media_player.clementine.markdown]: This looks like the LG logo.

**Comment by andrey-git** [comment]
Any updates on this PR?

**Comment by jjmontesl** [comment]
I squashed the pull requests and resubmitted. Let me know if there's something else I can do.

**Comment by andrey-git** [comment]
Looks fine, but lets wait for @fabaff

**Pull request merged**